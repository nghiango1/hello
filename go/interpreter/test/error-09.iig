// "divide by zero: 2 / 0",
2/0
