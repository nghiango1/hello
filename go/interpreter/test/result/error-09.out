ERROR: divide by zero: 2 / 0
